src/message/messageActionSheet.js

```diff
@@ -169,9 +169,12 @@ const unstarMessage = async ({ auth, message }) => {
 unstarMessage.title = 'Unstar message';
 unstarMessage.errorMessage = 'Failed to unstar message';
 
+export const messageFilter = (messageContent: string): string =>
```

Comment:
This name is much too generic for this context, and it doesn't actually describe this function's purpose, anyway. (This is a "filter" for the *content* of messages, rather than for the messages themselves; and in the following commit it becomes more than a filter.)